templates/whatson/whats_on_page.html

```diff
@@ -17,73 +17,69 @@
 
     <div class="tna-container">
       <div class="tna-column tna-column--full">
-        <form class="filters" method="GET" action="#">
-
+        <form class="filters" method="get" action="{% pageurl page %}">
           <fieldset class="filters__fieldset">
             <legend class="filters__super-title">When</legend>
             <div class="filters__panel">
-              <label for="event-date" class="filters__label filters__label--top">Choose a date</label>
+              <label for="{{ filter_form.date.id_for_label }}" class="filters__label filters__label--top">{{ filter_form.date.label }}</label>
               <p class="filters__help">Format: dd/mm/yyyy</p>
               {# we need a default value for the date in iOS or it is displayed tiny #}
-              <input type="date" name="event-date" id="event-date" class="filters__date" value="{% now 'Y-m-d' %}">
+              {{ filter_form.date }}
```

Comment:
Question: does this change break the display on iOS?

Reply:
I'm not too sure how I'd check for that, I only know how to change the screen size 😬 

I'm also not too sure how to add like a default value etc? This is my first time properly trying to manipulate the forms, I feel like there's a better way to do this but couldn't find it

Reply:
You can provide initial data either at the field level, or when you instantiate the form: https://docs.djangoproject.com/en/4.2/ref/forms/api/#initial-form-values

Reply:
I can't seem to get initial to work, I've tried it before like this:

```
date = forms.DateField(
        label="Choose a date",
        required=False,
        widget=forms.DateInput(attrs={"type": "date", "class": "filters__date"}),
        initial=timezone.now().date(),
    )
```

and tried formatting the date to be `dd/mm/yyyy` but that didn't work either